lib/pharos/yaml_file.rb

```diff
@@ -50,7 +50,7 @@ def read(variables = {})
 
     def erb_result(variables = {})
       Namespace.new(variables).with_binding do |ns_binding|
-        ERB.new(@content, nil, '%<>-').tap { |e| e.location = [@filename, nil] }.result(ns_binding)
+        ERB.new(@content, nil, '<>-').tap { |e| e.location = [@filename, nil] }.result(ns_binding)
```

Comment:
```suggestion
        ERB.new(@content, nil, '-').tap { |e| e.location = [@filename, nil] }.result(ns_binding)
```

`<>-` is equal to `-` as the code does: 

```ruby
if mode.include?('-')
  return [perc, '-']
end
```

Reply:
I think it can be one of:
```
%
%<
%<>
%-
```